services/venice-server/src/main/java/com/linkedin/venice/listener/StatsHandler.java

```diff
@@ -37,6 +36,10 @@ public StatsHandler(
     serverStatsContext = new ServerStatsContext(singleGetStats, multiGetStats, computeStats);
   }
 
+  public ServerStatsContext constructStatsContext() {
```

Comment:
Why do we need to build stats context here and out of `StatsHandler`? Why not pass it as dependency ?

Reply:
i guess we can... if we did so we would have to construct the context within listener service as we need the three aggregate stats. We also provide a method to create the stats context in order to be able to use this for gRPC functionality.